### DISCUSSION TITLE
sceptre default covariate model

### DISCUSSION THREAD

**Discussion by redbybeing** [Katsevich-Lab/sceptre#127] [Q&A] [answered]
Hi, 
I have a naive question for the default model formula (for high moi) used in sceptre.
`formula(~ log(response_n_nonzero) + log(response_n_umis) + log(grna_n_nonzero) + log(grna_n_umis) + response_p_mito + batch)`
I am curious why log is applied for the count values? I happened to try removing log from the formula and run and it did seem to increase the number of false positives in the negative control pairs. So I guess somehow log is a must for better calibration?

And, is the exact same formula used for all pairs tested? Would there be a case where some pairs are better fitted with another model?

Thank you!

Best,
Jiseok

**Comment by ekatsevi** [comment, marked as answer]
> I am curious why log is applied for the count values?

Our rationale is primarily for the sequencing depth variables like `response_n_umis`. To see why the logarithmic transformation makes sense at least for this variable, see Section 5.2.2 of [my lecture notes](https://katsevich-teaching.github.io/stat-9610-fall-2023/assets/lecture-notes.pdf). This section is in the context of Poisson regression, but it's similar for NB regression. Furthermore, the notes discuss adding a term like `log(response_n_umis)` as an *offset* (i.e. with coefficient 1), but in `sceptre` we add it as a *covariate* (i.e. with a coefficient we fit based on the data) for added robustness.

> And, is the exact same formula used for all pairs tested? Would there be a case where some pairs are better fitted with another model?

Yes, the same formula is used for all pairs tested. We have not come across a case yet where we need to use different formulas for different pairs.

**Reply by redbybeing** [reply]
Thank you so much for the kind explanations Eugene!
